dev-itpro/developer/devenv-data-transfer.md

```diff
@@ -138,6 +138,24 @@ begin
 end;
 ```
 
+### Example 2 - replace ModifyAll()
```

Comment:
Could you maybe move this part just after the section "Uniqueness in the source table" but before "CopyRows". It seems the "Performance" and "Uniqueness in the source table" sections pertains to the above example.

Reply:
Good point, updated.